travis/travis_install_nightly

```diff
@@ -233,4 +234,7 @@ if [ "$PG_LOGS_ENABLE" == "1" ]; then
     python $(dirname ${BASH_SOURCE[0]})/psql_log.py
 fi
 
+# Enable the python version
+source ${REPO_REQUIREMENTS}/virtualenv/python${TRAVIS_PYTHON_VERSION}/bin/activate
```

Comment:
Why we need this change here if the installation was on the top?

Reply:
Now, MQT shouldn't be very strict to use it just for travis or our image.
I mean, we are working to create a standard library to install package but this change don't help us to the focus.

Maybe we need to change T2D